### DISCUSSION TITLE
laravel-pdf vs. Browsershot?

### DISCUSSION THREAD

**Discussion by plweil** [spatie/laravel-pdf#11] [Q&A]
What's the difference between laravel-pdf and Browsershot?

**Comment by freekmurze** [comment]
Laravel PDF has a nice API for generating PDFs

**Comment by plweil** [comment]
The API does look like an improvement over Browsershot's , but the requirements of Laravel 10+ and PHP 8.2 are a showstopper for us - at least until we can get around to upgrading.

**Comment by its4nas** [comment]
> What's the difference between laravel-pdf and Browsershot?

Laravel-PDF is focused on generating PDF documents with advanced formatting and layout options, while Browsershot is more suitable for capturing web page content as PDF files.
